contrib/kafka/pkg/controller/channel/reconcile.go

```diff
@@ -117,9 +116,10 @@ func (r *reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 // boolean indicates if this Channel should be immediately requeued for another reconcile loop. The
 // returned error indicates an error during reconciliation.
 func (r *reconciler) reconcile(ctx context.Context, channel *eventingv1alpha1.Channel) (bool, error) {
+	var err error
```

Comment:
My preference is to not declare this variable, if we can avoid it. It looks we do need to declare it around line 133, but we don't need to do so here. So my preference is to remove this line, and use `:=` in other places (except line 133, which will need to declare it again).